pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3817,7 +3817,7 @@
         "4.0.2.0": "4.3.0"
       }
     },
-    "System.Runtime.Intrinsics.Experimental": {
+    "System.Runtime.Intrinsics": {
       "InboxOn": {},
```

Comment:
Do we need an "InboxOn" value yet? I didn't see any netcoreapp3.0 entries yet (we had removed netcoreapp2.1 in the rename to Experimental).

Reply:
Looks like we do. Updated.